app/src/components/common/RuleEditorModal/index.tsx

```diff
@@ -12,10 +12,10 @@ import {
 import { RQButton, RQEditorTitle, RQModal } from "lib/design-system/components";
```

Comment:
@RuntimeTerror10 Is this new TestThisRule UI compatible with the old extension version? 
Can you check once. If now we'll need to see how to support both

Reply:
Yes, new UI is compatible with the old extension version.